doc/bash-completion.sh

```diff
@@ -836,6 +836,10 @@ _restic_key()
     local_nonpersistent_flags+=("--help")
     flags+=("--new-password-file=")
     local_nonpersistent_flags+=("--new-password-file=")
+    flags+=("--user")
```

Comment:
This needs to be removed from the PR, the bash completion files are automatically generated for each release.